doc/dev/decisionrecords/Codestyle.md

```diff
@@ -1,96 +1,96 @@
-# Codestyle
+# Code Style
 
 We use the following code conventions for [Java](#Java) and [JavaScript](#JavaScript).
 
 ## Java
 
 The OpenTripPlanner Java code style is revised in OTP v2.2. We use the
-[Prettier Java](https://github.com/jhipster/prettier-java) as is. Maven is setup to
+[Prettier Java](https://github.com/jhipster/prettier-java) as is. Maven is set up to
 run `prettier-maven-plugin`. A check is run in the CI build, which fails the build preventing
-merging a PR if the code-style is incorrect.
+merging a PR if the code style is incorrect.
 
-There is two ways to format the code before checkin it in. You may run a normal build with maven -
-it takes a bit of time, but reformat the entire codebase. Only code you have changed should be
-formatted, since the existing code is already formatted. The second way is to set up prettier and
+There are two ways to format the code before checking it in. You may run a normal build with
+Maven—it takes a bit of time, but reformat the entire codebase. Only code you have changed should be
```

Comment:
When using the ASCII minus sign as a dash, I am more accustomed to seeing it surrounded by spaces. I see that here it has been replaced with an actual unicode em-dash character, but at least in my browser it is practically indistinguishable from the old-school conflated minus-hyphen-em-dash. I think even in cases where one can reliably render a true em-dash, some people still surround it with spaces. I propose that we sidestep the problem by not using em-dashes at all in technical documentation :) A semicolon or period might work here. The more pressing fix in this sentence would be to change the tense of "reformat".

Reply:
Thank you, that's a good point. In cases like this where we don't have control over the rendering, we should stick to ASCII characters whenever possible. Reverted and also added the "reformat" fix.